schemas/EGA.DAC.json

```diff
@@ -38,13 +38,15 @@
         "type": "string",
         "title": "Title of the DAC",
         "description": "Short free-form text that can be used to call out DAC records in searches or displays.",
+        "minLength": 1,
```

Comment:
What does this change do? Is it related to the addition of the submission object?

Reply:
It basically means that the given string needs to be of at least length 1. In other words, no empty string is allowed. 
As you mentioned it's not related to submission objects directly, just generally.